src/sentry/static/sentry/app/views/projectDebugFiles.jsx

```diff
@@ -134,9 +134,7 @@ class ProjectDebugSymbols extends AsyncComponent {
               {features &&
                 features.map(feature => (
                   <Tooltip key={feature} title={getFeatureTooltip(feature)}>
-                    <span>
-                      <Tag inline>{feature}</Tag>
-                    </span>
+                    <Tag inline>{feature}</Tag>
```

Comment:
Was there a reason `Tag`s needed the span?

Reply:
Not that I can see outside of anchoring the tooltip.

Reply:
I like this phrase, load bearing span. But I don't know what it means lol

Reply:
Loading bearing means something holding the structure up. Another usage would be "Don't knock that wall down it is load bearing". Might be 🇨🇦 vernacular.

Reply:
"load bearing" is definitely a phrase ai know, I just wasn't sure in what way a span could be load bearing haha